src/Adapters.Tests/EventSource.Netstandard13.Tests/EventSource.Netstandard13.Tests.csproj

```diff
@@ -17,7 +17,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
     <PackageReference Include="MSTest.TestAdapter" Version="1.1.13" />
     <PackageReference Include="MSTest.TestFramework" Version="1.1.13" />
-    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.4.0-beta1-build03597" />
+    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.4.0-beta1-build06467" />
```

Comment:
It is. But what I have effectively means "06467 or newer", which is what I want.